pedrovgs/HaveANiceDay
# Project bootstrap

### :pushpin: References
* **Issue:** Project bootstrap.

### :tophat: What is the goal?

Start the project using Finatra 😃 

### How is it being implemented?

We've followed the next steps to create this PR:

* Create an empty SBT project.
* Add [finatra](https://twitter.github.io/finatra/) as a core dependency.
* Configure a main class named ``HaveANiceDayServer`` we will use to initialize finatra.
* Add three files to improve contributions and used as templates to the repository so we can improve the quality of our pull requests by having a some checklists for issues, pull requests and other contributions.
* Configure [Travis CI support](https://travis-ci.org/pedrovgs/HaveANiceDay/builds).
* Configure [scalafmt](http://scalameta.org/scalafmt/) with a simple configuration.
* Update the project readme showing how to build run and test the project.

### How can it be tested?

Checkout this branch and run ``sbt run``. You'll see how the server starts and you can send a request to ``localhost:9000/`` and get an OK response. You can also execute ``sbt test`` and let the tests do this for you 😃
